src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -271,16 +271,15 @@ namespace winrt::TerminalApp::implementation
     };
 
     // Method Description:
-    // - Update our list of filtered actions to reflect the current contents of
+    // - Produce a list of filtered actions to reflect the current contents of
     //   the input box. For more details on which commands will be displayed,
     //   see `_getWeight`.
     // Arguments:
-    // - <none>
+    // - A collection that will receive the filtered actions
     // Return Value:
     // - <none>
-    void CommandPalette::_updateFilteredActions()
+    void CommandPalette::_collectFilteredActions(Windows::Foundation::Collections::IObservableVector<TerminalApp::Command>& actions)
```

Comment:
nit: since this function isn't really returning anything else, you might as well just return the vector out of this method, rather than passing it as an "out" parameter.